sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py

```diff
@@ -49,11 +49,13 @@ def _from_json(event, encode):
 
 class CloudEvent(EventMixin):   #pylint:disable=too-many-instance-attributes
     """Properties of an event published to an Event Grid topic using the CloudEvent 1.0 Schema.
+    Note: CloudEvents can only be sent to a topic of CloudEvent schema.
+    For EventGrid schema, use EventGridEvent.
 
     All required parameters must be populated in order to send to Azure.
 
     :param source: Required. Identifies the context in which an event happened. The combination of id and source must
-        be unique for each distinct event. If publishing to a domain topic, source must be the domain name.
+        be unique for each distinct event, and otherwise can be any user-defined string. If publishing to a domain topic, source must be the domain name.
```

Comment:
```suggestion
        be unique for each distinct event, and otherwise can be any user-defined string.
If publishing to a domain topic, source must be the domain name.
```